examples/search_examples/public/sql_search/app.tsx

```diff
@@ -0,0 +1,174 @@
+          }
+        },
+        error: (e) => {
+          setIsLoading(false);
+          if (e instanceof AbortError) {
+            notifications.toasts.addWarning({
+              title: e.message,
+            });
+          } else {
+            notifications.toasts.addDanger({
+              title: 'Failed to run search',
+              text: e.message,
+            });
+          }
+        },
+      });
+  };
+
+  return (
+    <EuiPageBody>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>SQL search example</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentBody>
+          <EuiForm>
+            <EuiFlexGroup>
+              <EuiFlexItem grow>
+                <EuiTextArea
+                  placeholder="SELECT * FROM library ORDER BY page_count DESC"
+                  aria-label="SQL query to run"
+                  value={sqlQuery}
+                  onChange={(e) => setSqlQuery(e.target.value)}
+                  fullWidth
+                  data-test-subj="sqlQueryInput"
+                />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiSuperUpdateButton
+                  isLoading={isLoading}
+                  isDisabled={sqlQuery.length === 0}
+                  onClick={doSearch}
+                  fill={true}
+                  data-test-subj="querySubmitButton"
+                />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiForm>
+
+          <EuiFlexGroup gutterSize="l">
+            <EuiFlexItem grow style={{ minWidth: 0 }}>
+              <EuiPanel grow>
+                <EuiText>
+                  <h3>Request</h3>
+                </EuiText>
+                <EuiCodeBlock
+                  language="json"
+                  fontSize="s"
+                  paddingSize="s"
+                  overflowHeight={720}
+                  isCopyable
+                  data-test-subj="requestCodeBlock"
+                  isVirtualized
+                >
+                  {JSON.stringify(request, null, 2)}
+                </EuiCodeBlock>
+              </EuiPanel>
+            </EuiFlexItem>
+            <EuiFlexItem grow style={{ minWidth: 0 }}>
+              <EuiPanel grow>
+                <EuiText>
+                  <h3>Response</h3>
+                </EuiText>
+                <EuiCodeBlock
+                  language="json"
+                  fontSize="s"
+                  paddingSize="s"
+                  isCopyable
+                  data-test-subj="responseCodeBlock"
+                  overflowHeight={720}
+                  isVirtualized
+                >
+                  {JSON.stringify(rawResponse, null, 2)}
+                </EuiCodeBlock>
+              </EuiPanel>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiPageContentBody>
+      </EuiPageContent>
+    </EuiPageBody>
+  );
+};
```

Comment:
How simple would it be to add a datagrid with the table results? I think it would be a nice addition. (Or maybe being able to swap the view between a datagrid and the raw response.)

Reply:
> (Or maybe being able to swap the view between a datagrid and the raw response.)

I think Lukas just meant here to improve UI of an example plugin?

Reply:
Sorry, meant to leave a comment here but GitHub wasn't cooperating at the time. Yes, I just meant improve the UI. We can take care of this in a separate PR if we decide it's helpful.